docs/index.adoc

```diff
@@ -105,10 +118,8 @@ For example, this concerns information how to build the final A2L file, if the F
 === A2L Description Files
 
 The A2L description depends on the FMU binary, for example, regarding memory addresses, and byte order.
-If an FMU archive contains multiple binaries for different platforms, the associated A2L files are placed into separate subdirectories below `/extra/org.fmi-standard.fmi-ls-xcp` following the same scheme as in the `binaries` directory, see <<Structure of the FMU archive>>.
-
-Note, that the A2L files placed under the `extra` directory are not accessible to the FMU at runtime.
-Any files that shall be accessible to the FMU at runtime must (also) be placed into the `resources` directory as defined by the FMI standard.
+If an FMU archive contains multiple binaries for different platforms, the associated A2L files are placed into separate subdirectories below `/extra/org.fmi-standard.fmi-ls-xcp` and in the binaries folder, see <<Structure of the FMU archive>>.

```

Comment:
@PenndorfT the binaries folder should not be involved. Also the need to use the resource folder has been discussed e.g. with @andreas-junghanns. Let's not start reverting each other's changes online but discuss it in the next design meetings.